src/qml/PositioningSettings.qml

```diff
@@ -45,4 +45,5 @@ Settings {
   property int digitizingMeasureType: 1
 
   property bool geofencingPreventDigitizingDuringAlert: false
+  property bool enableEgeniouss: false
```

Comment:
```suggestion
  property bool egenioussEnabled: false
```
But for `positioningActivated` we use Active instead of Enable. But in the code we see generally more often "Enabled". Sot not sure which one you should choose :smile: